meta-flex-distro/conf/distro/flex-os.conf

```diff
@@ -373,6 +379,9 @@ include conf/distro/include/flex-ml.conf
 # BSP related distro changes
 include conf/distro/include/flex-distro.conf
 
+# EWAOL integration
+require ${@bb.utils.contains('USER_FEATURES', 'flex-ewaol-baremetal', 'conf/distro/include/ewaol-flex-integration.conf', '', d)}
```

Comment:
We use DISTRO_FEATURES usually instead of USER_FEATURES in such conditions. Functionally it's fine, just for consistency.

Reply:
This is distro conf. DISTRO_FEATURES are still being parsed, and feature_overrides.bbclass works on event bb.event.ConfigParsed so this user feature wouldn't have made it to DISTRO_FEATURES yet. There are other such cases too e.g. https://github.com/MentorEmbedded/meta-flex-security/blob/develop/meta-flex-selinux/conf/distro/include/flex-security-selinux.conf#L10

Reply:
Interesting. Perhaps worth mentioning somewhere in comments or commit message.

Reply:
Added a comment